### PR TITLE
CDK L1s profile update: Auditing, Alert, Project, Cluster,DbUser,Serverless

### PR DESCRIPTION
**Pull request by SuperMohit**
## Description

CDK L1s profile update: Auditing, Alert, Project, Cluster,DbUser,Serverless

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change works in Atlas

### CODE REVIEW

File: cdk/cdk-resources/alert-configuration/src/index.ts [hunk 1]
```diff
@@ -27,6 +22,13 @@ export interface CfnAlertConfigurationProps {
    */
   readonly eventTypeName?: CfnAlertConfigurationPropsEventTypeName;
 
+  /**
```
Comment (andreaangiolillo) [inline cdk/cdk-resources/alert-configuration/src/index.ts:25]: Doesn't the alert configuration still have the readOnly issue?

Comment (andreaangiolillo) [reply cdk/cdk-resources/alert-configuration/src/index.ts:25]: if the answer is yes, I believe this constructor will still need changes before releasing it

**Review by andreaangiolillo — DISMISSED** [review]
LGTM. Comments is not blocking

File: cdk/cdk-resources/alert-configuration/src/index.ts (hunk 1 above)
Comment (SuperMohit) [reply cdk/cdk-resources/alert-configuration/src/index.ts:25]: yes. sure. But the changes should first flow through CFN. But Rn, it aligns with what is published.

**Comment by SuperMohit** [comment]
Project, Cluster, DbUser, and ip-access-list have been published.

**Pull request merged**